orbstation/modules/cargo/cadavers.dm

```diff
@@ -0,0 +1,125 @@
+	contains = list(/mob/living/carbon/human/species/ethereal)
+
+// Remove the crystal core, both so the ethereal doesn't regenerate and so the crew can't bulk-order crystal cores
+/datum/supply_pack/medical/cadaver/ethereal/generate()
+	. = ..()
+	var/mob/living/carbon/human/corpse = locate() in .
+	for (var/obj/item/organ/internal/heart/ethereal/illegal_heart in corpse.internal_organs)
+		qdel(illegal_heart)
+
+/datum/supply_pack/medical/cadaver/felinid
+	name = "Medical Cadaver Crate (Felinid)"
+	desc = "A preserved medical cadaver, for dissections, spare organs, or experiments. This one is a felinid."
+	contains = list(/mob/living/carbon/human/species/felinid)
+
+/datum/supply_pack/medical/cadaver/human
+	name = "Medical Cadaver Crate (Human)"
+	desc = "A preserved medical cadaver, for dissections, spare organs, or experiments. This one is a human."
+	contains = list(/mob/living/carbon/human)
+
+/datum/supply_pack/medical/cadaver/jelly
+	name = "Medical Cadaver Crate (Jellyperson)"
+	desc = "A preserved medical cadaver, for dissections, spare organs, or experiments. This one is a jellyperson."
+	contains = list(/mob/living/carbon/human/species/jelly)
+
+/datum/supply_pack/medical/cadaver/moth
+	name = "Medical Cadaver Crate (Moth)"
+	desc = "A preserved medical cadaver, for dissections, spare organs, or experiments. This one is a moth."
+	contains = list(/mob/living/carbon/human/species/moth)
+
+/datum/supply_pack/medical/cadaver/pod
+	name = "Medical Cadaver Crate (Podperson)"
+	desc = "A preserved medical cadaver, for dissections, spare organs, or experiments. This one is a podperson."
+	contains = list(/mob/living/carbon/human/species/pod)
+
+/datum/supply_pack/medical/cadaver/rat
+	name = "Medical Cadaver Crate (Ratfolk)"
+	desc = "A preserved medical cadaver, for dissections, spare organs, or experiments. This one is a ratfolk."
+	contains = list(/mob/living/carbon/human/species/ratfolk)
+
+/datum/supply_pack/medical/cadaver/tiziran
+	name = "Medical Cadaver Crate (Tiziran)"
+	desc = "A preserved medical cadaver, for dissections, spare organs, or experiments. This one is a tiziran."
+	contains = list(/mob/living/carbon/human/species/lizard)
+
+// I'm making this a subtype of coffin solely so vampires can sleep in them.
+/obj/structure/closet/crate/coffin/stasis
+	name = "stasis coffin"
+	desc = "An advanced coffin used to keep corpses fresh for extended transport. Once opened, it will permanently stop functioning."
+	icon = 'orbstation/icons/obj/coffin.dmi'
+	icon_state = "stasiscoffin"
+	resistance_flags = NONE
+	max_integrity = 70
+	material_drop = /obj/item/stack/sheet/iron
+	material_drop_amount = 5
+	open_sound = 'sound/machines/closet_open.ogg'
+	close_sound = 'sound/machines/closet_close.ogg'
+	open_sound_volume = 25
+	close_sound_volume = 50
+	can_install_electronics = FALSE
+	mob_storage_capacity = 1 //no mass stasis, sorry
+	var/functional = TRUE //whether the coffin still works or not
```

Comment:
Not a big deal but this being above as a triple slashed comment would be preferable (like `vampire_arrived` and `vampire_chance`), that format makes it show up in vscode intellisense when you hover over the variable.